src/features/fireWeather/components/WxDataGraphToggles.tsx

```diff
@@ -29,14 +32,18 @@ const WxDataToggles = ({
   setShowReadings,
   noModels,
   showModels,
-  setShowModels
+  setShowModels,
+  noForecasts,
+  showForecasts,
+  setShowForecasts
 }: Props) => {
   const classes = useStyles()
 
   return (
     <FormGroup row>
       <FormControlLabel
         className={classes.formControlLabel}
+        data-testid="show-readings-toggle"
```

Comment:
These newly added 3 test ids are not being used anywhere, did you intend to test them?

Reply:
oops. that's leftover from a while ago when I was trying to make up more tests to get >80% coverage. I'll delete the IDs now - I'm not using them and I don't think they're needed anymore